packages/backend/src/config/discovery/arbitrum.ts

```diff
@@ -19,3 +27,7 @@ const arbitrumRawConfig = createConfigFromTemplate({
 
 const arbitrumDiscoveryConfig = new DiscoveryConfig(arbitrumRawConfig)
 const arbitrumEventsToWatch = getEventsToWatch(addresses)
+const arbitrumChangelogWhitelist = toEthereumAddresses([
+  addresses.ultraLightNodeV2,
+  addresses.endpoint,
+])
```

Comment:
I think we also want to include changes in the msig

Reply:
https://github.com/l2beat/lz-monitoring/pull/56/commits/45b50157501d0f47936e0001a2cf853a1f3fb273